ISISComputingGroup/EPICS-lakeshore
# Lakeshore 336 units warnings

https://github.com/ISISComputingGroup/IBEX/issues/1760

The db file had two definitions for the PVs listed in the ticket (e.g. "$(P):HEATER1:RANGE"). This was because one of the template files had a second definition of these records which just provided string descriptions for the values.
Removing these fixed the problem - I am at a bit of a loss why these existed, perhaps DLS do something clever with them?

To test:

- Rebuild the support module with the changes
- Rebuild the IOC
- Confirm that the PVs listed in the ticket only appear once in the db file

Hopefully, the DbChecker will confirm this too!
